tractdb_pyramid/views/storytelling_stravaview.py

```diff
@@ -38,19 +38,12 @@ def get_runs(request):
 
     # Be sure it exists
     if not admin.exists_document('strava_access_token'):
-        # James Says: This should probably be an error rather than a not found
-        request.response.status_int = 404
+        request.response.status_int = 403
```

Comment:
At first I thought maybe this should be a 500, but decided a 403 is ok because the person has not given us permission to access their Strava data.